### PR TITLE
Import test model from string

### PR DESCRIPTION
**Pull request by st34k**

### CODE REVIEW

File: packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts [hunk 1]
```diff
@@ -7,14 +8,27 @@ import { importSchema } from 'graphql-import';
...
 const parser = DefaultParser.create(DatabaseType.postgres);
-const dataModelPath = config.has('graphql.schema.datamodel.path') ? config.get('graphql.schema.datamodel.path') : null;
 
-export const openCrudDataModel = dataModelPath && parser.parseFromSchemaString(fs.readFileSync(dataModelPath, 'utf8'));
+let openCrudDataModel;
+let openCrudSchemaGraphql;
+
+if (config.has('graphql.schema.datamodel.path')) {
+  const dataModelPath = config.get('graphql.schema.datamodel.path');
+  openCrudDataModel = dataModelPath && parser.parseFromSchemaString(fs.readFileSync(dataModelPath, 'utf8'));
```
Comment (brafdlog) [inline packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts:20]:
```suggestion
  openCrudDataModel = parser.parseFromSchemaString(fs.readFileSync(dataModelPath, 'utf8'));
```

File: packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts [hunk 2]
```diff
@@ -7,14 +8,27 @@ import { importSchema } from 'graphql-import';
 import { DefaultParser, DatabaseType } from 'prisma-datamodel';
 import { makeExecutableSchema } from 'graphql-tools';
 import { graphqlSync, introspectionQuery } from 'graphql';
+import { sdl, datamodel } from '../test/models';
```
Comment (brafdlog) [inline packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts:11]: To make the code clear that these are test models
```suggestion
import * as testModels from '../test/models';
```

File: packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts [hunk 3]
```diff
@@ -7,14 +8,27 @@ import { importSchema } from 'graphql-import';
...
+  openCrudDataModel = dataModelPath && parser.parseFromSchemaString(fs.readFileSync(dataModelPath, 'utf8'));
+} else {
+  openCrudDataModel = parser.parseFromSchemaString(datamodel);
+}
+
+if (config.has('graphql.schema.sdl.path')) {
+  const schemaSdlPath = config.get('graphql.schema.sdl.path');
+  openCrudSchemaGraphql = schemaSdlPath && importSchema(schemaSdlPath);
+} else {
+  openCrudSchemaGraphql = importSchema(sdl);
```
Comment (brafdlog) [inline packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts:29]: Does this work on a string sdl? I thought from their api that it works only on a file path an that we will need to do parser.parseFromSchemaString here

Comment (st34k) [reply packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts:29]: yep, actually the other way around (parseSchemaString) failed

**Pull request merged**